src/pcp2json/pcp2json.1

```diff
@@ -547,6 +555,14 @@ determines the actual time
 will run.
 .RE
 .TP
+\fB\-u\fR, \fB\-\-url\fR
```

Comment:
There's a section for 'pcp2json specific options' in the man page describing how these should be in the config file, I think these should be mentioned there as well.

Reply:
Fixed, thanks.